game/resource/addon_english.txt

```diff
@@ -280,164 +280,164 @@
-        "Script_4_Wards_8" "ok, let's put a ward on this high ground before we just walk up there to our deaths. Select the ward in your inventory with the left mouse button, then click on the high ground target. This will place the ward. You don't need to be very close to do it!"
-        "Script_4_Wards_9" "well done! But whats this? There is no one here! This cant be right, we scanned this area and it said someone was here? Are the scans lying? Is nothing real?"
+        "Script_4_Wards_8" "OK, let's put a ward on this high ground before we just walk up there to our deaths. Select the ward in your inventory with the left mouse button, then click on the high ground target. This will place the ward. You don't need to be very close to do it!"
+        "Script_4_Wards_9" "Well done! But what's this? There is no one here! This can't be right, we scanned this area and it said someone was here? Are the scans lying? Is nothing real?"
         "Script_4_Wards_10" "Calm down friend. Now go pick up that odd blue ward."
         "Script_4_Wards_11" "Notice that it combines with your other ward in your inventory. Please double click on your ward in your inventory"
-        "Script_4_Wards_12" "this blue ward is called a sentry ward. It’s job is to show invisible things. This includes wards, and other units that are invisible! Place your blue sentry ward down on top of your yellow observer ward, just like you did last time. Click on the ward with the left mouse button, and then click on the map to place it in on the target."
+        "Script_4_Wards_12" "This blue ward is called a sentry ward. Its job is to show invisible things. This includes wards, and other units that are invisible! Place your blue sentry ward down on top of your yellow observer ward, just like you did last time. Click on the ward with the left mouse button, and then click on the map to place it in on the target."
         "Script_4_Wards_13" "HOLY SHIT"
         "Script_4_Wards_14" "AH HA! There he is! Go and attack him! Click on him with your right mouse button!"
-        "Script_4_Wards_15" "oh nooo, he left the area of your sentry! Hold alt to see the area of effect your sentry has."
+        "Script_4_Wards_15" "Oh nooo, he left the area of your sentry! Hold 'ALT' to see the area of effect your sentry has."
         "Script_4_Outpost_1" "Looks like we will need to hunt him down. Here is a dust of appearance!"
-        "Script_4_Outpost_2" "Pick that up, and walk in the direction you saw the riki"
+        "Script_4_Outpost_2" "Pick that up, and walk in the direction you saw Riki."
         "Script_4_Outpost_3" "Now hit the dust and hope he's close!"
         "Script_4_Outpost_4" "There he is! Get that pile of garbage"
         "Script_4_Outpost_5" "Oh no, we lost him again. Maybe this time for good. Oh, one last idea. Let's take that enemy outpost!"
```

Comment:
Optional: Change "outpost" to "Outpost". If changed here, should be changed on all other sentences with this word.